browsers/webview_android.json

```
@@ -245,10 +245,7 @@
           "release_date": "2019-01-29",
           "release_notes": "https://chromereleases.googleblog.com/2019/01/chrome-for-android-update_29.html",
           "status": "current"
-        },
-        "73": {
-          "status": "nightly"
-        }
+        
```

Whoops, syntax error!

```suggestion
        }
```